setup.py

```diff
@@ -0,0 +1,18 @@
+"""A setuptools based setup module.
+
+See:
+https://packaging.python.org/guides/distributing-packages-using-setuptools/
+https://github.com/pypa/sampleproject
+"""
+
+from setuptools import setup, find_packages
+
+setup(
+	name='attending',
+	version='0.0.1',
+	description='A manager for offline documentation',
+	url='https://github.com/ivanov/attending',
+	packages=find_packages(exclude=['fizbuz', 'foobar', 'latest', 'prior-work', 'tests', 'versionless']),
```

Comment:
Perhaps we can fix this list after the folders move.

Reply:
you can also do `packages=['attending']` and skip find_packages altogether.